web/src/constants/index.ts

```diff
@@ -1 +1,3 @@
-export {}
\ No newline at end of file
+export const CURRENCY_TYPE = '$';
+export const ITEMS_ON_PAGE = 12;

```

Comment:
Actually items on page and best sellers count is a unieque Constant and should be in the file we you them. You can revert constants.ts and move variables back